dashboard/config/initializers/devise.rb

```diff
@@ -315,6 +315,20 @@
     manager.failure_app = CustomDeviseFailure
   end
 
+  Warden::Manager.after_authentication do |user, auth|
+    if auth.cookies[:pm] == "new_header"
```

Comment:
thanks Will.  I had a similar thought, but I think it's reasonable to tie this to a feature that happens to have a UI component and also a cookie usage.  And I think definitely worthwhile so that we don't give every production user a cookie that we later realise we want to change.  (Which I think happened with our original `language` cookie, which is why we now use `language_`)